### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by Katherinemair** [community/community#68004] [GitHub Education]
### Hi everyone,

Hi there 

I have the approval for the GitHub Student Developer Pack but still waiting for the benefits. Would you mind having a look. 

![image](https://github.com/community/community/assets/145578437/7d3075b8-e493-4c2f-a658-215af7e0513f)

**Comment by arafajs** [comment]
Hi @Katherinemair 
According to the GitHub documentation, the activation process of Benefits for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by Katherinemair** [reply]
I will wait for a couple of days more !! Hopefully is coming soon !!

**Comment by mantasu** [comment]
Hey, I had the same issue, you just have to wait for a few more days as there will be some time before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).
